core/class/jMQTTCmd.class.php

```diff
@@ -246,6 +246,36 @@ public function preSave() {
 
         }
 
+		// Reset auto_publish if info cmd
+		if ($this->getType() == 'info' && $this->getConfiguration('auto_publish', 0))
+			$this->setConfiguration('auto_publish', 0);
+		// Check "request" value if auto_publish enabled
+		if ($this->getType() == 'action' && $this->getConfiguration('auto_publish', 0)) {
```

Comment:
maybe check that 'request' in configuration changed

Reply:
Indeed, done in next commit